packages/ui/src/hooks/use-social-callback-handler.ts

```diff
@@ -8,8 +8,23 @@ const useSocialCallbackHandler = () => {
 
   const socialCallbackHandler = useCallback(
     (connectorId: string) => {
+      // Get search string to evaluate
+      const searchString = window.location.search
+
+      // Get hash string to evaluate
+      const hashString = window.location.hash
+
+      // Define evaluated search string
+      let search = searchString
+
       // Apple use fragment mode to store auth parameter. Need to support it.
-      const search = window.location.search || '?' + window.location.hash.slice(1);
+      if (!searchString && hashString) {
+        search = `?${hashString.slice(1)}`
+      }
+
+      if (searchString && hashString) {
+        search = `${search}&${hashString.slice(1)}`
+      }
```

Comment:
We intend to avoid mutable let variables.  How about
```
const search = `${searchString ?? '?'}${hashString.slice(1)}`
```

Reply:
P.S. the commit lint ci failed. 

`fix(callback): Added search + hash string interpolation`
The subject can not start with a uppercase I believe

Reply:
> We intend to avoid mutable let variables. Please check the following lint error:
> 
> <img alt="image" width="776" src="https://user-images.githubusercontent.com/36393111/224882985-89ac0480-e970-4f9c-b617-435a3538bca9.png">
> 
> How about
> 
> ```
> const search = `${searchString ?? '?'}${hashString.slice(1)}`
> ```

Won't really work when both are present.
- Outcome will be ?a=xb=y. <-- No `&` between x as value and b as parameter naming.

I'm suggesting:

```typescript
`${searchString ?? '?'}${(searchString && hashString ? '&' : '')}${hashString.slice(1)}`
```

Advantages:
- Keeps initial use case intact. <-- Either search or hash, if either is present.
- Combines both, if both are expected in conjunction.

Reply:
Commit lint rules are resolved as well.
Thanks!